R/process_phenotypes.R

```diff
@@ -0,0 +1,342 @@
+#' Resolve strain names to isotypes
+#'
+#' \code{resolve_isotypes} takes a vector of strain names and converts them to CeNDR-defined isotype names
+#'
+#' @param strains2resolve a vector of strain names to change to isotype names.
+#' @param isotype_lookup a tibble that contains two columns - strain, previous_name, isotype.
+#' This is used to change input strain names. Default option queries CeNDR isotypes
+#' @return Output is a vector with isotype names
+#' @export
+resolve_isotypes <- function(strains2resolve,
```

Comment:
It is rare to see intention as part of a function parameter.

Reply:
whatchyu mean?

Reply:
The name `strains2resolve` is a format not often seen and describes what will happen to the variable. That's not what you usually see. It should probably be just `strains`, the function name describes what is going to happen. The current format could makes the parameter name inconsistent with other functions and therefore more confusing.

Reply:
ah, gotchyu

Reply:
done